Cargo.toml

```diff
@@ -175,12 +175,12 @@ wasm-bindgen-test = { version = "0.3.37", default-features = false, features = [
 libc = { version = "0.2.148", default-features = false }
 
 [build-dependencies]
-cc = { version = "1.0.83", default-features = false }
+cc = { version = "1.2.8", default-features = false }
 
 # At least 1.0.93 is requried for visionOS, but some versions around that point
 # have bugs that seem to have been fixed in 1.0.97 or so.
-[target.'cfg(all(target_vendor = "apple", target_os = "visionos"))'.build-dependencies]
-cc = { version = "1.0.97", default-features = false }
+# [target.'cfg(all(target_vendor = "apple", target_os = "visionos"))'.build-dependencies]
+# cc = { version = "1.0.97", default-features = false }
```

Comment:
You can just remove lines 180-183 as they are no longer doing anything given your change above.